src/App.tsx

```diff
@@ -206,7 +218,7 @@ export class App extends React.Component {
             <Hotkey key={2} group={regionGroupTitle} global={true} combo="shift + l" label="Unlock all regions" onKeyDown={this.unlockAllRegions}/>,
             <Hotkey key={3} group={regionGroupTitle} global={true} combo="del" label="Delete selected region" onKeyDown={appStore.deleteSelectedRegion}/>,
             <Hotkey key={4} group={regionGroupTitle} global={true} combo="backspace" label="Delete selected region" onKeyDown={appStore.deleteSelectedRegion}/>,
-            <Hotkey key={5} group={regionGroupTitle} global={true} combo="esc" label="Deselect region" onKeyDown={appStore.deselectRegion}/>,
+            <Hotkey key={5} group={regionGroupTitle} global={true} combo="esc" label="Deselect region/Untoggle region creation mode" onKeyDown={this.handleRegionEsc}/>,
```

Comment:
"Untoggle" sounds a little weird. What if we just say "Deselect region / Cancel region creation" ?